sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,157 @@
+| 1st                | 876,434        | Dec 2024             | 1,558,786,847    | 500 (50%)  | 3.37%          |
+| 2nd                | 1,086,818      | Dec 2028             | 1,663,978,847    | 250 (50%)  | 1.58%          |
+| 3rd                | 1,297,202      | Dec 2032             | 1,716,574,847    | 125 (50%)  | 0.77%          |
+| -                  | 2,197,560      | Jan 2050 (17.08y)    | 1,829,119,597    | 125 (0%)   | 0.36%          |
+
+
+The _proposed_ STX emission schedule is presented as follows.  In particular, this SIP proposes preserving the 1000 STX coinbase until April 2026.  After this, there would be a halving to 500 STX after two years, aligning the remaining Stacks halving block heights with Bitcoin's halving schedule. Four years later, emissions halve again to 125 STX, and the tail emission after the final halving in 2036 would be 62.5 STX. This brings the total supply in 2050 to 1,804,075,347, or about 0.77% lower than the original 1,818,000,000 estimate.
+
+| Coinbase Reward Phase | Bitcoin Height | Approx Date           | STX Supply at Block | STX Reward | Annual Inflation |
+|--------------------|----------------|---------------------|-------------------|------------|-----------------|
+| Current            | 870,100        | -                   | 1,552,452,847    | 1000       | -               |
+| 1st                | 945,000        | Apr 2026 (+1.33y)   | 1,627,352,847    | 500 (50%)  | 3.23%          |
+| 2nd                | 1,050,000      | Apr 2028 (2y)       | 1,679,852,847    | 250 (50%)  | 1.57%          |
+| 3rd                | 1,260,000      | Apr 2032 (4y)       | 1,732,352,847    | 125 (50%)  | 0.76%          |
+| 4th                | 1,470,000      | Apr 2036 (4y)       | 1,758,602,847    | 62.5 (50%)   | 0.37%          |
+| -                  | 2,197,560      | Jan 2050 (13.83y)   | 1,804,075,347    | 62.5 (0%)   | 0.18%          |
+
+
+With these changes, miner incentives and PoX yield remain unchanged for another 16 months, which we believe is sufficient time for the nascent Nakamoto signer set to develop into a set of stable, professional block signers, and for the sBTC project to attract sufficient initial liquidity.
+
+The model for both the current and proposed emissions schedules can be found in the SIP-029 Stacks Emission Model[4].
+
+# Related Work
+
+While many blockchain protocols implement token emission schedules with periodic reductions in block rewards, we are not aware of any precedent for modifying an emission schedule specifically to maintain economic incentives during major protocol upgrades.
+
+# Backwards Compatibility
+
+This change requires a hard fork of the Stacks blockchain
+
+# Activation
+
+## Voting Timeline
+
+Voting will begin at bitcoin block height 870,750, which occurs ~ Sunday, November 17th, 2024.
+Voting will conclude at bitcoin block height 872,750, which occurs ~ Sunday, December 1st, 2024.
```

Comment:
When does this new schedule activate?  Can we get a target activation height?

I'd propose 876,000, which is about 22 days after voting closes.